src/consensus/aft/raft.h

```diff
@@ -614,8 +629,13 @@ namespace aft
     // guaranteed that we will eventually receive all of them, since all
     // nodes keep re-submitting ORCs until they are able to switch to the next
     // pending configuration.
-    bool orc(kv::ReconfigurationId rid, const ccf::NodeId& node_id) override
+
+    std::optional<kv::Configuration::Nodes> orc(
+      kv::ReconfigurationId rid, const ccf::NodeId& node_id) override
     {
+      // TODO: What about lock?
```

Comment:
This is called from the endpoint and if I remember correctly, the lock is already held there, at least when we're in BFT mode. But then, BFT may be going away anyways...

Reply:
Good point. It looks like it works with the lock in CFT mode so let's see.